File: Arnav Sharma/Assignment-0/Loops/sum_odd.java
public class sum_odd {
    public static void main(String[] args) {
        int sum = 0;
        for (int i = 1; i <= 10; i++) {
            if (i % 2 != 0) {
                sum = sum + i;
            }
        }
        System.out.println("Sum of all odd from 1 to 10 is: " + sum);
    }
    
}

File: Sawtantar/Loops/sumodd.cpp
#include <iostream>

using namespace std;

int main()  
{  
    int num, count = 1, sum = 0;  
  
   cout<<"Enter the limit: "; 
    cin>>num; 
  
    while(count <= num)  
    {  
        if(count%2 != 0)  
        {  
            sum = sum + count;  
        }  
        count++;  
    }  
    cout<<"Sum of odd numbers are: "<<sum;  
    return 0;  
}  